packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -647,6 +651,10 @@ class SvgParser {
 
     newRoot = opacityPeepholeOptimizer.apply(newRoot);
 
+    if (enableMaskingOptimizer && path_ops.isPathOpsInitialized) {
```

Comment:
I think we should throw here if enableMaskingOptimizer is true and `!path_ops.isPathOpsInitialized`.

Reply:
fixed.